apps/vessel-history/features/search/search.selectors.ts

```diff
@@ -4,6 +4,8 @@ import { selectAdvancedSearchFields, selectQueryParam } from 'routes/routes.sele
 import { getSerializedQuery } from './search.thunk'
 
 export const selectVesselsFound = (state: RootState) => state.search.queries
+export const selectSearchError = (state: RootState) => state.search.error
+export const selectSearchSources = (state: RootState) => ['AIS']
```

Comment:
Why is this selector returning always `['AIS']`? shouldn't it get the sources from the API response?

Reply:
ups, I added to test, nice catch kkk